### ISSUE TITLE
change mbox_source_layers

### ISSUE THREAD

**Issue by liangxi627** [lzx1413/CAFFE_SSD#3]
Hello, when I use the FSSD, I changed the mbox_source_layers and have a error when training my own dataset, and the training loss decline fast at first, finally maintain about 5%, how can I solve it? 
> F0512 14:40:32.193830 15163 solver.cpp:464] Check failed: result[j]->width() == 5 (1 vs. 5)

**Comment by lzx1413** [comment]
can you run the training process on the VOC dataset successfully?

**Comment by liangxi627** [comment]
Yes, it runs successfully on VOC, I meet the same error when I use the original SSD codes.

**Comment by lzx1413** [comment]
It seems like that this error occurs while testing the model, the test result should have 5 elements rather than 1. So maybe there is something wrong with the dataset for testing?

**Comment by liangxi627** [comment]
Ok, thank you, maybe I will check the dataset. And I found SSD is hard to training on small objects,  is there any suggestions for  the hyperparameters  setting?

**Comment by lzx1413** [comment]
1. Set the min_size and max_size for priorbox generation with appropriate values for your objects.
2. Use feature map with feature stride 4 may be helpful to small objects.

**Comment by liangxi627** [comment]
Many thanks for your prompt help!
